src/understanding-deis/concepts.md

```diff
@@ -11,27 +12,52 @@ We consider it an invaluable synthesis of much experience with
 software-as-a-service apps in the wild, especially on the
 Heroku platform.
 
-Deis is designed to run applications that adhere to [Twelve-Factor App][]
+Workflow is designed to run applications that adhere to [Twelve-Factor App][]
 methodology and best practices.
 
+## Kubernetes
+
+[Kubernetes][] is an open-source cluster manager developed by Google and
+donated to the [Cloud Native Compute Foundation][cncf]. Kubernetes manages all
+the activity on your cluster. Including converging to a desired state,
+providing critical functions like stable service addresses, health monitoring,
+service discovery and DNS resolution.
+
+Workflow builds upon Kubernetes abstractions like Services, Replication
+Controllers and Pods to provide a developer-friendly UX, source to image, log
+aggregation, etc.
+
+Workflow is shipped as a Kubernetes-native application, installable via
+[Helm][helm]. So operators familiar with Kubernetes will feel right at home
+running Workflow.
+
+For a detailed overview of Workflow components, see our [component][components] break down.
+
 ## Docker
 
-[Docker][] is an open source project to pack, ship and run any
+[Docker][] is an open source project to build, ship and run any
 application as a lightweight, portable, self-sufficient container.
 
-Deis curates your applications as Docker images, which are then
-distributed across your cluster as Docker containers.
+Workflow uses Docker Images and the Docker Engine (via Kubernetes) to package
+and run your application respectively.
```

Comment:
I'm assuming build-pack apps are the average case.  If so, this sentence might not be technically correct in the majority of cases since such apps are never built into an image, but rather, merely a slug.  It's certainly a minor distinction, but improved accuracy seldom has hurt anyone.